### PR TITLE
Drop all but prometheus source

### PR DESCRIPTION
**Pull request by lberk**
Fixes: knative/community#237

## Proposed Changes

- Drop everything but the prometheus bits
- Move eventing-contrib/prometheus to eventing-prometheus

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,25 +1,13 @@
...
-- Integrate with AWS SQS
-- Integrate with Ceph
-- Integrate with GitHub
-- Integrate with GitLab
-- Integrate with NATS Streaming
-- Integrate with Prometheus
-- Integrate with Websockets
-- Expose an ingress
+The Knative Eventing Prometheus Event Source enables Knative Eventing integration with the
+[Prometheus Monitoring System](https://prometheus.io/).
```
Comment (mattmoor) [inline README.md:10]: Format markdown:
```suggestion
The Knative Eventing Prometheus Event Source enables Knative Eventing
integration with the [Prometheus Monitoring System](https://prometheus.io/).
```

**Review by mattmoor — COMMENTED** [review]
Produced via:
  `prettier --write --prose-wrap=always $(find -name '*.md' | grep -v vendor | grep -v .github | grep -v docs/cmd/)`

**Comment by lberk** [comment]
@lionelvillard would you mind taking a look?

**Pull request merged**